external/coreclr/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
This was already in coreclr.depproj. I'm OK if you go ahead and create the configurations.props for this and for netstandard20. I will try and stop fighting the use of configuration.props :)